docs/modules/ROOT/pages/appendix.adoc

```diff
@@ -41,12 +41,12 @@ We do support the following processing instructions:
 * `<?assert followed by a valid precondition ?>`
 * `<?assume followed by a valid precondition ?>`
 
-Look up valid preconditions <<concepts_preconditions,here>>. The full XMl schema for <<concepts_migrations_catalog-based, catalog-based migrations>> looks like this:
+Look up valid preconditions xref:concepts.adoc#concepts_preconditions[here]. The full XMl schema for xref:concepts.adoc#concepts_migrations_catalog-based[catalog-based migrations] looks like this:
```

Comment:
It's recommended to use the `xref` macro instead of `<<` `>>`.

Since we are using reference to the same module (ROOT), the same documentation component (migrations) and the same version (unversioned) we can use only the file name `concepts.adoc`.

Please note that you can reference pages from other labs components. For instance, you could create a reference to a page of the Kafka Integration Documentation:

```
xref:4.1@kafka:ROOT:kafka-connect.adoc[]
```

https://docs.antora.org/antora/latest/navigation/xrefs-and-link-text/#page-xrefs